src/app/public/modules/type-definitions/anchor-link.service.ts

```diff
@@ -41,6 +41,9 @@ export class SkyDocsAnchorLinkService {
       const regex = createRegex(typeName);
 
       let matches: RegExpExecArray;
+      let counter = 0;
+      const max = 100;
```

Comment:
Added a safeguard just in case another anchor-replacement bug is found (otherwise the page freezes and potentially crashes the browser).